src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -42,10 +47,12 @@ class VirtualMachine(
     }
 
     fun runProgram(): Pair<ExitStatus, List<ManimInstr>> {
-        linearRepresentation.add(CodeBlock(displayCode, codeBlockVariable, codeTextVariable, pointerVariable))
+        linearRepresentation.add(PartitionBlock("1/3", "2/3"))
```

Comment:
Will this mess with the way the algorithm allocates space for data structures on the RHS? I thought the ratio was 5/14 : 9/14

Reply:
I'm not 100% sure but it seems to look fine for programs that have been tried. The partition block makes frames for the different subsections so if those are being used hopefully it works fine

Reply:
Yeah I think because it's 1/3 the code block will actually be tiny bit smaller than it can be, but at least it's smaller over larger